widget_scripts.md

```diff
@@ -1,3 +1,112 @@
 
 # Widget scripts
 
+### Widgets (HORUS ONLY) Scripts
+
+##### General description
+
+Widgets are small scripts that show some info in a 'zone' in one of the model specific user defined (telemetry) screens. You can define those screens within the telemetry menu on the HORUS. 
+
+Each model can have up to five custom screens, with up to 8 widgets per screen, depending on their size and layout. Each instance of a widget has his own custom settings.
+
+##### File Location
+
+Widgets are located on the SD card, each in their specific folder /WIDGETS/&lt;*name*&gt;/main.lua (*name* must be in 8 characters or less).
+
+##### Lifetime of telemetry script
```

Comment:
telemetry? Should that be widget script?

Reply:
yes, tried to be consequent and missed this one.

Reply:
By the way, me neither. LOL